index.html

```diff
@@ -2033,6 +2051,99 @@ <h3 id="applying">
           </p>
         </section>
       </section>
+
+      <section>
+        <h3>
+          <code>serviceworker</code> member
+        </h3>
+        <p>
+          The <dfn><code>serviceworker</code> member</dfn> describes a
+          service worker as defined in [[!SERVICE-WORKERS-1]].
+        </p>
+        <p>
+          The <a><code>serviceworker</code> member</a> represents a <a
+          href="https://w3c.github.io/ServiceWorker/#service-worker-registration-concept">
+          service worker registration</a>. Other service worker registrations can be done, for instance
+          by a script; if these have different scopes they will be considered separate
+          registrations. If they have the same scope and script URL, they coalesce.
+          If they have different script URLs, last one wins.
+        </p>
+        <p>
+          The <dfn>steps for processing the <code>serviceworker</code>
+          member</dfn> are given by the following algorithm. The algorithm
+          takes a <a>manifest</a> <var>manifest</var>. This algorithm returns a
+          registration object <var>registration</var>, which can be <code>undefined</code>.
+        </p>
+        <ol>
+          <li>Let <var>unprocessed registration</var> be the result of calling the
+          <a>[[\GetOwnProperty]]</a> internal method of <var>manifest</var>
+          with argument "<code>serviceworker</code>".
+          <li>Let <var>src</var> be the result of running the <a>steps
+          for processing the <code>src</code> member of a service worker</a>
+          with <var>unprocessed registration</var> and <var>manifest URL</var>.
+          </li>
+          <li>If <var>src</var> is <code>undefined</code>, or if the result of
+          running <a>Is origin potentially trustworthy</a>
+          with the origin of <var>src</var> is <code>Not Trusted</code>, abort
+          these steps and return <code>undefined</code>.
+          </li>
+          <li>Otherwise, let <var>registration</var> be an object with
+          properties <code>src</code>, <code>scope</code>, <code>type</code> and
+          <code>use_cache</code>. All properties initially set to
+          <code>undefined</code>.
+          </li>
+          <li>Set <var>registration</var>'s <code>src</code> property to be
+          <var>src</var>.
+          </li>
+          <li>Let <var>scope</var> be the result of running the <a>steps
+          for processing the <code>scope</code> member of a service worker</a>
```

Comment:
<li>If <a>obtaining the manifest</a> succeeds, and the
              <var>service worker registration</var> of <a>manifest</a> is not
              <code>undefined</code>, a user agent can at this point...

It is not invoked if it fails